e2e/cypress/tests/integration/enterprise/system_console/channel_moderation/helpers.ts

```diff
@@ -1,13 +1,19 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {ChannelType} from '@mattermost/types/lib/channels';
+
+import {Matcher} from '@testing-library/dom';
+
+import {UserProfile} from '@mattermost/types/users';
+
 import * as TIMEOUTS from '../../../../fixtures/timeouts';
 import {getAdminAccount} from '../../../../support/env';
 
 import {checkBoxes} from './constants';
 
 // # Visits the channel configuration for a channel with channelName
-export const visitChannelConfigPage = (channel) => {
+export const visitChannelConfigPage = (channel: { id?: string; name: string; create_at?: number; update_at?: number; delete_at?: number; team_id?: string; type?: ChannelType; display_name?: string; header?: string; purpose?: string; last_post_at?: number; last_root_post_at?: number; creator_id?: string; scheme_id?: string; teammate_id?: string; status?: string; group_constrained?: boolean; shared?: boolean; props?: Record<string, unknown>; policy_id?: string }) => {
```

Comment:
@adithyaakrishna , 
just wondering if Cypress.Channel would work.

Reply:
@yasserfaraazkhan I guess it will, I will send a commit up to update it soon